persistence/sql/identity/persister_identity.go

```diff
@@ -1129,6 +1129,9 @@ func (p *IdentityPersister) DeleteIdentity(ctx context.Context, id uuid.UUID) (e
 	return nil
 }
 
+// This function is only used internally to cleanup partially created identities,
+// when creating a batch of identities at once and some failed to be fully created.
+// This act should not be observable externally and thus we do not emit an event.
```

Comment:
In Go comments that document the function go outside the function and start with the function name.

As this reads like an internal comment, it should be moved to the function body.

Also, at some point we wanted to expose batch delete to the API, but maybe it was premature to make it public here ;).